testsuites/repository/src/main/java/org/eclipse/rdf4j/testsuite/repository/TupleQueryResultTest.java

```diff
@@ -35,28 +31,17 @@
 import org.eclipse.rdf4j.rio.RDFParseException;
 import org.eclipse.rdf4j.rio.UnsupportedRDFormatException;
 import org.eclipse.rdf4j.sail.SailException;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public abstract class TupleQueryResultTest {
 
 	private final Logger logger = LoggerFactory.getLogger(TupleQueryResultTest.class);
 
-	@BeforeClass
-	public static void setUpClass() throws Exception {
-		System.setProperty("org.eclipse.rdf4j.repository.debug", "true");
-	}
-
-	@AfterClass
-	public static void afterClass() throws Exception {
-		System.setProperty("org.eclipse.rdf4j.repository.debug", "false");
-	}
-
```

Comment:
Rewrote this test to use Junit 5. I know that it's a public class in our Test Suite so I'm not sure if this would be backwards compatible or not. If not then we can create a new package and copy my rewrite over to that package and deprecate the old class.